keyboards/toffee_studio/blueberry/info.json

```diff
@@ -0,0 +1,372 @@
+{
+    "keyboard_name": "Blueberry",
+    "manufacturer": "Toffee Studio",
+    "url": "",
+    "maintainer": "Toffee Studio",
+    "usb": {
+        "vid": "0x1067",
+        "pid": "0x626C",
+        "device_version": "0.0.1"
+    },
+    "processor": "atmega32u4",
+    "bootloader": "atmel-dfu",
+    "diode_direction": "COL2ROW",
+    "layouts": {
+        "LAYOUT": {
```

Comment:
this looks to be pretty close to 65_ansi?

Reply:
it is yeah;
https://toffeestudio.com/
https://assembly.toffeestudio.com/ (1. Getting Started \ Choosing a Layout)

Reply:
If it matches the layouts in /layouts/default/, then support for those should be added. 

https://docs.qmk.fm/#/feature_layouts

Reply:
it doesnt match the layout its just pretty close to it, so adding a feature layout may not work completely